src/Joomlatools/Console/Command/Site/AbstractSite.php

```diff
@@ -41,14 +41,26 @@ protected function configure()
             "Web server root",
             '/var/www'
         )
+        ->addOption(
+            'use-webroot-dir',
+            null,
+            InputOption::VALUE_NONE,
+            "Uses directory specified with --www as the site install dir"
+         )
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->site       = $input->getArgument('site');
         $this->www        = $input->getOption('www');
-        $this->target_dir = $this->www.'/'.$this->site;
+        echo "WR OPTION ".$input->getOption('use-webroot-dir');
```

Comment:
@levnikmyskin Could you remove the `echo` statements please? I guess there are leftovers from debugging? :)

Reply:
Ops, sorry. I thought I removed them all, but it looks like this one's still there